src/rust/Dockerfile

```diff
@@ -6,6 +6,8 @@ SHELL ["/bin/bash", "-c"]
 
 RUN apt-get update && apt-get install -y --no-install-recommends \
         wait-for-it \
+        ca-certificates \
+        netbase \
```

Comment:
These seem unnecessary for the build image, we already have them in the deploy base. Can these be removed?